vault/resource_okta_auth_backend.go

```diff
@@ -79,14 +83,18 @@ func oktaAuthBackendResource() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    false,
 				Optional:    true,
+				Default:     "0",
```

Comment:
What does 0 mean for ttl and max_ttl?

Reply:
It's not completely clear from the docs what a value of 0 means, only that it is the default https://www.vaultproject.io/api-docs/auth/okta#token_ttl

Reply:
Used a computed value from Vault instead of the default.